nbs/07_nlp.external.transformers-glue.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Are we sure that <code>one_batch</code>  is needed? Have you tried on the lastest fastai? It shouldn't be needed anymore with https://github.com/fastai/fastai/commit/5bc369c27846819edb825253fbb7bec0213cc2cb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/walkwithfastai/walkwithfastai.github.io/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>